packages/react/src/components/builder-content.component.tsx

```diff
@@ -227,21 +229,43 @@ export class BuilderContent<
     const TagName = this.props.dataOnly ? NoWrap : 'div'
 
     return (
-      <TagName
-        {...(!this.props.dataOnly && {
-          ref: ref => (this.ref = ref)
-        })}
-        className="builder-content"
-        onClick={this.onClick}
-        builder-content-id={useData && useData.id}
-        builder-model={this.props.modelName}
-      >
-        {this.props.children(
-          useData && useData.data,
-          this.props.inline ? false : loading,
-          useData
-        )}
-      </TagName>
+      <VariantsProvider isStatic={this.props.isStatic} initialContent={useData}>
```

Comment:
looking on at the tracking calls here, I think I should've added the variants provider on top of the builderContent in BuilderPage... I'll update